lens/database.ml

```diff
@@ -63,7 +63,10 @@ let fmt_cols ~db f cols =
 let fmt_phrase_value ~db f v =
   Format.fprintf f "%s"
     ( match v with
-    | LPV.Bool b -> string_of_bool b
+    | LPV.Bool b -> (
+        match b with
+        | true -> "(1=1)"
```

Comment:
The fairly old `sqlite3` version used by our CI does not support these constants which is indeed depressing. This method is luckily portable, but it might be nice to change it back to the old code once `true` and `false` become more widespread.

Reply:
Yes, we have the same issue with shredding (`row_number` not supported by Sqlite3 < 3.25).  On one hand it is good to support old versions, on the other hand the default ubuntu 16 version of Sqlite3 is pretty old.  I think we should manually install a newer version in the CI and recommend people use the most recent version.